app/(main)/_components/item.tsx

```
@@ -0,0 +1,71 @@
+}
+
+export const Item = ({
+    id,
+    label,
+    onClick,
+    icon: Icon,
+    documentIcon,
+    isSearch,
+    expanded,
+    active,
+    onExpand,
```

ESLint

[error] 'onExpand' is defined but never used.

(@typescript-eslint/no-unused-vars)